### DISCUSSION TITLE
Minor Auto-correct entry fix : password

### DISCUSSION THREAD

**Discussion by ghost** [OpenBangla/OpenBangla-Keyboard#343] [General]
the word password is suggested as পাসস্বরদ by default and there is no auto-correct entry for it
I manually auto-corrected it to pasowoyarrD as it then outputs as পাসওয়ার্ড fine

it would be nice if you added this to the default auto-correct entries

**Comment by Dyrimon** [comment]
'পাসওয়ার্ড' is written follwoing the Avro keyboard layout. Use " pas\`OyarrD ", as the capital D is for ড and d is for দ. Use the backticks symbol(\`) to separate -কার from consonant. But of course if you are setting an auto-correct for convenience then you don't have to remember the Avro layout.
